podcasts/PlaybackItem.swift

```diff
@@ -15,9 +15,17 @@ class PlaybackItem: NSObject {
 
     func createPlayerItem() -> AVPlayerItem? {
         guard let url = EpisodeManager.urlForEpisode(episode) else { return nil }
-
-        let customHeaders = [ServerConstants.HttpHeaders.userAgent: ServerConstants.Values.appUserAgent]
-        let asset = AVURLAsset(url: url, options: ["AVURLAssetHTTPHeaderFieldsKey": customHeaders])
+        var options = [String : Any]()
+        if #available(iOS 16, *) {
```

Comment:
The code doesn't compile because `AVURLAssetHTTPUserAgentKey` is only available for watch OS extension 9+, we need to do this check too.

```suggestion
        if #available(iOS 16, *), #available(watchOSApplicationExtension 9.0, *) {
```